src/components/Input/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import {
+  Input as Field,
+  Label as FieldLabel,
+  styles,
+  themeColor,
+  Typography,
+  themeSpacing,
+} from '@datapunt/asc-ui';
```

Comment:
I miss a input placeholder here

Reply:
Not sure what you mean. There is no `Placeholder` component in `@datapunt/asc-ui`. If you mean setting a placeholder on the input element, that is already possible by passing that prop, like so
```jsx
  <Input placeholder="Foo bar baz" />
```

Line 44 takes care of that.

Reply:
yes true but I the test is still <Input placeholder="Foo bar baz" /> missing